e2e_playwright/st_dataframe_selections_test.py

```diff
@@ -121,6 +121,24 @@ def test_single_row_select(app: Page):
     )
 
 
+def test_single_row_select_with_sorted_column(app: Page):
+    canvas = _get_single_row_select_df(app)
+
```

Comment:
I believe it would make sense to add a pre-sort check as well, to validate that the sorting changes something and its not accidentally matching.
- Click on first row -> save output
- Keep the rest of the test but also validate that the new selection != the selection from the previous point

Reply:
Added this to the test case 👍